Sources/KituraNet/HTTP/HTTPServerConfiguration.swift

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright IBM Corporation 2019
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+public struct HTTPServerConfiguration {
```

Comment:
The naming and facilities on this struct need to be consistent between Kitura-net and Kitura-NIO.  In https://github.com/IBM-Swift/Kitura-net/pull/307 I named this `ServerOptions`.  I don't have a strong preference as to which name we go with, however you'll need to change this anyway as there is a facility for sending a custom response back to the client which would need replicating here.